Combat.md

```diff
@@ -1,5 +1,63 @@
+
+### Initiative and Turn Order
+When deciding who goes first in a [Round], have everyone roll 1d20 and add their initiative. By default, everyone takes actions in order from highest to lowest - break ties with a re-roll.
+
+You can always choose to treat your initiative as lower than what you rolled.
+
+You and an ally can swap positions in the initiative order at the beginning of combat. Whoever rolled higher takes the initiative score of the other. Whoever rolled lower takes the higher initiative score - 2, or their initiative score + 1, whichever is lower. _Example: Alice rolls a 17 and Bob rolls a 5. They swap, so Alice has a 5 and Bob has a 15. Example: Alice rolls a 12 and Bob rolls a 10. They swap. Alice has a 10 and Bob has an 11._
+
+### On Your Turn
+At the beginning of your turn, you lose any AP you did not use last [Round] and gain 3 new AP. You can then begin taking actions, spending AP as appropriate. The following table lists action types and their AP cost. The next section lists some common combat actions.
+
+|Action Type|AP Cost|
+|---|---|
+|[Instant Action]|0 AP|
+|[Fast Action]|1 AP|
+|[Standard Action]|2 AP|
+|[Slow Action]|3 AP|
+|[Immediate Action]|0 AP*|
+|[Reaction]|0 AP*|
+
+__[Immediate Actions] and [Reactions]:__ These special kinds of actions can be taken when it is not your turn. Anything that is an [Immediate Action] or [Reaction] will list the precise conditions under which it can be taken. While you can take any number of [Immediate Actions], many GMs will limit this to a reasonable number (say, 3). However, you can only ever take a single [Reaction] per [Round]. 
+
+## Common Actions
+
+### Standard Attack
+Also known as "attack." This is a [Standard Action]. Select one of your weapons you are currently wielding and an opponent you can target with that weapon. Make an attack roll. If you are using a martial weapon, you have to equal or exceed their AC to hit. If you are using a magical weapon, you have to equal or exceed their SR to hit (and use a spell attack roll).
+
+_Attack roll = 1d20 + Strength + weapon bonuses + misc. bonuses_  
+_Spell Attack Roll = 1d20 + Intelligence + weapon bonuses + misc. bonuses_
+
+If the natural result of the d20 - the number that lands facing up - is equal to or greater than your weapons critical threshold, you scored a critical hit. If the attack roll still was not high enough to hit your opponent, you still hit but it is only a regular hit.
+
+If your opponent currently has any [Miss Chances], check those now. Roll d%. If you roll less than or equal to their [Miss Chance], you hit. Otherwise, you miss. If you have a critical hit but would miss, you still hit but it is only a regular hit.
```

Comment:
```suggestion
If your opponent currently has any [Miss Chances], check those now. Roll d%. If you roll less than or equal to their [Miss Chance], you hit. Otherwise, you miss. If you have a critical hit that would miss from [Miss Chances], it becomes a regular hit instead of a miss.
```